src/components/MovieListPage/MovieListPage.tsx

```diff
@@ -112,54 +111,23 @@ export function MovieListPage() {
     setShowDialog(true);
   }
 
-  function handleAddMovie() {
-    setDialogContent({
-      title: 'EDIT MOVIE',
-      children: (
-        <MovieForm
-          onSubmit={() => {
-            setShowDialog(false);
-          }}
-        ></MovieForm>
-      ),
-    });
-    setShowDialog(true);
-  }
-
   return (
     <MovieListPageContainer>
       <TopContainer>
-        {selectedMovie ? (
-          <DetailsContainer>
-            <TopContainerHeader>
-              <AppLogo />
-              <SearchSwitcherButton onClick={() => setSelectedMovie(undefined)}>
-                <Icon src={SearchLogoIcon} />
-              </SearchSwitcherButton>
-            </TopContainerHeader>
-            <MovieDetails movie={selectedMovie}></MovieDetails>
-          </DetailsContainer>
-        ) : (
-          <SearchMovieContainer>
-            <TopContainerHeader>
-              <AppLogo />
-              <AddMovieButton onClick={handleAddMovie}>
-                + ADD MOVIE
-              </AddMovieButton>
-            </TopContainerHeader>
-            <SearchMovieContent>
-              <SearchForm initialValue={searchTerm} onSearch={setSearchTerm} />
-            </SearchMovieContent>
-          </SearchMovieContainer>
-        )}
+        <Outlet />
       </TopContainer>
 
       <MoviesContainer>
         <DetailsHeader>
-          <GenreSelect genreList={GENRE_LIST_OPTIONS} onSelect={setGenre} />
+          <GenreSelect
+            initialSelectedGenre={genre || ''}
+            genreList={GENRE_LIST_OPTIONS}
+            onSelect={setGenre}
+          />
           <SortControl
+            initialValue={sortBy}
             sortList={SORT_OPTIONS}
-            onSortChange={setSort}
+            onSortChange={(sortOption) => setSortBy(sortOption)}
```

Comment:
Sinde param and value is the same - we can simply call it like `onSortChange={setSortBy}`